js/header.js

```diff
@@ -1,4 +1,5 @@
-const headerShrinkWidth = 1290;
+const headerShrinkWidth = 1319;
+var miniTitle = document.getElementById('title-mini');
 header = document.getElementsByTagName('header')[0];
```

Comment:
Could you also add a const here? Don't know why we have nothing currently

Reply:
when i tested it as a const, it came up with an error due to another js file having header as well. I assume that we have it not set to const so that we could use the same variable name. I can change the name in one of the files if that is preferable.

Reply:
Ah, seems it's being declared here: https://github.com/frc1418/frc1418.github.io/blob/50e6e81dbbb0d2a460cc069148d2a2a6516a03eb/js/scripts.js

Could you remove that declaration and then use const here? It doesn't seem to be used in the file.

Reply:
I'll do that

Reply:
I'll do that